definitions/ads_domain/google_ads/views/ad.sqlx

```diff
@@ -39,4 +39,5 @@ SELECT DISTINCT
   ad_group_ad_status AS ad_status
 FROM (
 ${union}
-)
\ No newline at end of file
+)
+QUALIFY ROW_NUMBER() OVER (PARTITION BY CAST(customer_id AS STRING)) = 1
```

Comment:
Hi @martenlindblad , review that we must preserve the right hierarchy:
* 1 ad_id, belongs to 1 adgroup_id, belongs to 1 campaign, belongs to 1 account.
* 1 account, has n campaigns, which has n adgroups, which has n ads.

Please, add the other ids into your partition by .

Reply:
I think this is what you mean, please correct me otherwise!
`QUALIFY ROW_NUMBER() OVER(PARTITION BY account_id, campaign_id, adgroup_id, creative_id) = 1`

Reply:
CORRECT.